javascript/src/components/messageloaders/RecentMessageLoader.jsx

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var React = require('react/addons');
+var InputDropdown = require('../inputs/InputDropdown');
+var InputsStore = require('../../stores/inputs/InputsStore');
+
+var RecentMessageLoader = React.createClass({
+    getInitialState() {
+        return {
+            inputId: undefined
```

Comment:
Is this state ever used? I see you use it on the render method, but it is never set (or I can't see it).